charts/operator/templates/manager/controller-manager-metrics-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
```

Comment:
can we add common labels on all operator templates

Reply:
@pgvishnuram @danielhoherd @rishkarajgi  Can you suggest some common labels we can put for the operator components.? 
I am thinking of these, are these good enough? :
```
        component: (can have component name like metrics, rbac,cert,crds)
        tier: astronomer
        release: {{ .Release.Name }}
        version: {{ .Chart.Version }}
```